plugin/ochttp/trace.go

```diff
@@ -71,37 +70,36 @@ func (t *traceTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 
 	span.SetAttributes(responseAttrs(resp)...)
+	span.SetStatus(status(resp.StatusCode))
```

Comment:
This means that for unknown codes we set zero status. Instead, I think we should set status "2" for any HTTP response code >= 400. It's better to indicate an error vaguely than to indicate success. We should also add the exact error code as an attribute.

Reply:
We decided to set errors for anything outside of [200, 400). I will update the PR.